src/urllib3/poolmanager.py

```diff
@@ -62,6 +63,11 @@
     "key_server_hostname",  # str
 )
 
+# Version 3.7 introduces the parameter "blocksize" to
```

Comment:
```suggestion
# Python 3.7 introduces the parameter "blocksize" to
```

(Writing "Python 3.7" makes it easier to grep.)

Reply:
I have updated my fork. Thanks